AaSystem/EventQueue/CommandQueue.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+import queue
+
+
+commandQueue = "commandQueue"
+tempQueue = "tempQueue"
+
+
+CommandQueues = {
```

Comment:
what about classes?

Reply:
Most likely moving back to single queue implementation